model/PieAPPv0pt1_PT.py

```diff
@@ -8,64 +8,65 @@
 import skimage.io as io
 
 class PieAPP(nn.Module): # How to ensure that everything goes on a GPU? do I need to fetch?
-	def __init__(self,batch_size,num_patches):
-		super(PieAPP, self).__init__()
-		self.conv1 = nn.Conv2d(3,64,3,padding=1)
-		self.conv2 = nn.Conv2d(64,64,3,padding=1)
-		self.pool2 = nn.MaxPool2d(2,2)
-		self.conv3 = nn.Conv2d(64,64,3,padding=1)
-		self.conv4 = nn.Conv2d(64,128,3,padding=1)
-		self.pool4 = nn.MaxPool2d(2,2)
-		self.conv5 = nn.Conv2d(128,128,3,padding=1)
-		self.conv6 = nn.Conv2d(128,128,3,padding=1)
-		self.pool6 = nn.MaxPool2d(2,2)
-		self.conv7 = nn.Conv2d(128,256,3,padding=1)
-		self.conv8 = nn.Conv2d(256,256,3,padding=1)
-		self.pool8 = nn.MaxPool2d(2,2)
-		self.conv9 = nn.Conv2d(256,256,3,padding=1)
-		self.conv10 = nn.Conv2d(256,512,3,padding=1)
-		self.pool10 = nn.MaxPool2d(2,2)
-		self.conv11 = nn.Conv2d(512,512,3,padding=1)
-		self.fc1_score = nn.Linear(120832, 512)
-		self.fc2_score = nn.Linear(512,1)
-		self.fc1_weight = nn.Linear(2048,512)
-		self.fc2_weight = nn.Linear(512,1)
-		self.ref_score_subtract = nn.Linear(1,1)
-		self.batch_size = batch_size
-		self.num_patches = num_patches	
+    def __init__(self,batch_size,num_patches):
+        super(PieAPP, self).__init__()
+        self.conv1 = nn.Conv2d(3,64,3,padding=1)
+        self.conv2 = nn.Conv2d(64,64,3,padding=1)
+        self.pool2 = nn.MaxPool2d(2,2)
+        self.conv3 = nn.Conv2d(64,64,3,padding=1)
+        self.conv4 = nn.Conv2d(64,128,3,padding=1)
+        self.pool4 = nn.MaxPool2d(2,2)
+        self.conv5 = nn.Conv2d(128,128,3,padding=1)
+        self.conv6 = nn.Conv2d(128,128,3,padding=1)
+        self.pool6 = nn.MaxPool2d(2,2)
+        self.conv7 = nn.Conv2d(128,256,3,padding=1)
+        self.conv8 = nn.Conv2d(256,256,3,padding=1)
+        self.pool8 = nn.MaxPool2d(2,2)
+        self.conv9 = nn.Conv2d(256,256,3,padding=1)
+        self.conv10 = nn.Conv2d(256,512,3,padding=1)
+        self.pool10 = nn.MaxPool2d(2,2)
+        self.conv11 = nn.Conv2d(512,512,3,padding=1)
+        self.fc1_score = nn.Linear(120832, 512)
+        self.fc2_score = nn.Linear(512,1)
+        self.fc1_weight = nn.Linear(2048,512)
+        self.fc2_weight = nn.Linear(512,1)
+        self.ref_score_subtract = nn.Linear(1,1)
+        self.batch_size = int(batch_size)
+        self.num_patches = int(num_patches)
 
-	def flatten(self,matrix): # takes NxCxHxW input and outputs NxHWC
-		return matrix.view((self.batch_size*self.num_patches,-1))

```

Comment:
should be: 

`return matrix.view((self.batch_size*self.num_patches,-1))`

Reply:
Was it hitting an error about non-contiguous tensors?